File: bank-of-models/src/main/java/ro/fmi/bnk/models/BranchLocationModel.java
package ro.fmi.bnk.models;

public class BranchLocationModel {
		
	private String name;
	private Double longitude;
	private Double latitude;
	
	public BranchLocationModel(String name, Double longitude, Double latitude) {
		super();
		this.name = name;
		this.longitude = longitude;
		this.latitude = latitude;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public Double getLongitude() {
		return longitude;
	}
	public void setLongitude(Double longitude) {
		this.longitude = longitude;
	}
	public Double getLatitude() {
		return latitude;
	}
	public void setLatitude(Double latitude) {
		this.latitude = latitude;
	}
}

File: bank-of-web/src/main/webapp/src/app/primary-page/account/models/branchLocationModel.ts

	
export class BranchLocationModel {

     name: String;
     long:Float64Array;
     lat:Float64Array;


	
    constructor( name: String, long: Float64Array, lat: Float64Array) {
        this.name = name;
        this.long = long;
        this.lat = lat;

    }

    static fromJSONArray(array: Array<Object>): BranchLocationModel[] {
        return array["data"].map(obj => new BranchLocationModel(obj['name'], obj['longitude'], obj['latitude']));
    }
}